### DISCUSSION TITLE
Stripe Connect App Fee Issue

### DISCUSSION THREAD

**Discussion by billybobthornton1995** [pretix/pretix#3353] [Q&A]
I am having an issue with stripe connect and the app fee.

For an event in the stripe payment provider settings I entered the testing api keys of an account created via stripe connect. In the global settings I have entered the stripe connect secret and publishable testing api keys and left the live keys blank. I have also set stripe connect app fee (percent) to 5. When I make a test purchase for the event the test purchase goes through for the stripe connect child account but the app fee does not come through on the parent account.

There is also a setting for stripe connect client ID in global settings. Stripe has a client ID for their test mode and live mode but the global settings page in pretix only has one setting for client ID so I tried both as well as leaving it blank.

**Comment by joelebeau9** [comment]
Did you get this issue resolved Bob?
